cassandra/cluster.py

```diff
@@ -2124,14 +2124,14 @@ def prepare(self, query, custom_payload=None):
         future = ResponseFuture(self, message, query=None, timeout=self.default_timeout)
         try:
             future.send_request()
-            query_id, column_metadata, pk_indexes = future.result()
+            query_id, bind_metadata, pk_indexes, result_metadata = future.result()
```

Comment:
In the future, if we extend that result part again, we might prefer to create a structure for the future result attributes.